src/plugins/intel_cpu/src/transformations/cpu_opset/common/pass/split_fc.cpp

```diff
@@ -0,0 +1,102 @@
+        if (rt_info.count("parallelDomain")) {
+            return false;
+        }
+
+        // get input
+        auto src_item = fc_node->get_input_node_shared_ptr(0);
+        auto wgt_item = fc_node->get_input_node_shared_ptr(1);
+        bool weight_is_dynamic = wgt_item->is_dynamic();
+        if (weight_is_dynamic) {
+            return false;
+        }
+
+        auto out_shape = fc_node->get_output_partial_shape(0);
+
+        // split weight
+        constexpr size_t split_dim = 0; // split happens on the first dimension.
+        constexpr size_t split_num = 2; // split the tensor into two parts.
+        auto split_dim_range = wgt_item->get_shape()[split_dim];
+
+        // needn't to split fc when the dim is less than 1.
+        if (split_dim_range <= 1) {
+            return false;
+        }
+
+        auto split_dim_node = std::make_shared<ov::opset8::Constant>(ov::element::i64, ov::Shape{}, split_dim);
+
+        // We should use Split for even dim and VariadicSplit for odd dim.
+        std::shared_ptr<Node> split_wgts;
+        if (split_dim_range % 2 == 0) {
+            split_wgts = std::make_shared<ov::opset12::Split>(wgt_item,
+                                                              split_dim_node,
+                                                              split_num);
+        } else {
+            auto fc0_dim0 = static_cast<int64_t>((split_dim_range + 1) / 2);
+            auto fc1_dim0 = static_cast<int64_t>((split_dim_range - 1) / 2);
+            auto split_length = ov::opset8::Constant::create<int64_t>(ov::element::i64, ov::Shape{2}, {fc0_dim0, fc1_dim0});
+            split_wgts = std::make_shared<ov::opset12::VariadicSplit>(wgt_item,
+                                                                      split_dim_node,
+                                                                      split_length);
+        }
+
+        // sub fc
+        auto fc_output_type = fc_node->get_output_element_type(0);
+        const auto out_rank = out_shape.rank();
+        auto fc_node0 = std::make_shared<ov::intel_cpu::FullyConnectedNode>(src_item,
+                                                                            split_wgts->output(0),
+                                                                            out_rank,
+                                                                            fc_output_type);
+        auto fc_node1 = std::make_shared<ov::intel_cpu::FullyConnectedNode>(src_item,
+                                                                            split_wgts->output(1),
+                                                                            out_rank,
+                                                                            fc_output_type);
+        // runtime parallel
+        fc_node0->get_rt_info()["parallelDomain"] = fc_node->get_friendly_name();
+        fc_node1->get_rt_info()["parallelDomain"] = fc_node->get_friendly_name();
```

Comment:
In common case, it is not guaranteed that node friendly name is unique (despite the fact that we try to avoid duplications, it's "legal" from `ov::Model` perspective). Can it be a problem if several FullyConnected "numa groups" will have the same `parallelDomain` value? If yes, maybe it's better to store unique node name (we can get it via `get_name()` node method) in `parallelDomain`?